core/src/main/java/bisq/core/dao/governance/proposal/compensation/CompensationProposalService.java

```diff
@@ -62,11 +62,10 @@ public CompensationProposalService(BsqWalletService bsqWalletService,
 
     public ProposalWithTransaction createProposalWithTransaction(String name,
                                                                  String link,
-                                                                 Coin requestedBsq,
-                                                                 String bsqAddress)
+                                                                 Coin requestedBsq)
             throws ValidationException, InsufficientMoneyException, TxException {
         this.requestedBsq = requestedBsq;
-        this.bsqAddress = bsqAddress;
+        this.bsqAddress = "B" + bsqWalletService.getUnusedAddress().toBase58();
```

Comment:
Probably should have convenience functions toBSQAddress() and fromBSQAddress()

Reply:
Yes, good idea.